### DISCUSSION TITLE
Mouse?

### DISCUSSION THREAD

**Discussion by dining-philosopher** [omnideconv/immunedeconv#78] [Q&A] [answered]
Hello!

Is it possible to use immunedeconv on murine data?

Thanks in advance.

**Comment by grst** [comment, marked as answer]
@LorenzoMerotto is currently looking into mouse methods and the plan is to add them to immunedeconv eventually (See also https://github.com/icbi-lab/immunedeconv/issues/76). 
Lorenzo, could you maybe comment when you roughly expect this to be ready? 

In the meanwhile, you could try to translate the murine gene symbols to human orthologs and run immunedeconv with the human methods. I haven't tested it, but I would expect it to work reasonably well, especially for the marker-based methods (xCell and MCP counter). Alternatively, you can just apply the methods described in #76 directly until we have a wrapper ready in immunedeconv.
